cli/tests/info_recursive_imports_test.out

```diff
@@ -1,11 +1,12 @@
 local: [WILDCARD]info_recursive_imports_test.ts
 type: TypeScript
 deps: 4 unique (total [WILDCARD])
+
 file://[WILDCARD]cli/tests/info_recursive_imports_test.ts ([WILDCARD])
 └─┬ file://[WILDCARD]cli/tests/recursive_imports/A.ts ([WILDCARD])
   ├─┬ file://[WILDCARD]cli/tests/recursive_imports/B.ts ([WILDCARD])
   │ ├─┬ file://[WILDCARD]cli/tests/recursive_imports/C.ts ([WILDCARD])
   │ │ ├── file://[WILDCARD]cli/tests/recursive_imports/A.ts *
-  │ │ └── file://[WILDCARD]cli/tests/recursive_imports/common.ts ([WILDCARD])
```

Comment:
I am not fond of this change. Already displayed dependencies should be marked with *.

Reply:
It does, I was just finding that with the new module graph and circular dependencies, it wasn't deterministic which one got "seen" first.  It was a crap shoot.  I can try to figure out how to make it deterministic.

Reply:
I tried, but it seems that the recursion and the new `ModuleGraph2` don't like to be deterministic.  One of the three versions of `common.ts` gets marked with its size, and the other 2 get marked with the ` *`, it just seems like the luck of the draw.

Let me know how much time you want me to try to figure it out, because the information is accurate, it just isn't deterministic.